assets/sass/06-components/widgets.scss

```diff
@@ -96,6 +97,10 @@
 			background-color: var(--button--color-background);
 			color: var(--button--color-text);
 		}
+
+		@include media(mobile-small) {
+			margin-top: 5px;
```

Comment:
This added margin needs to apply to larger screen sizes as well. It isn't kicking in on the iPhone 8 or XS for example: 

<img width="375" alt="Screen Shot 2020-10-12 at 8 58 35 AM" src="https://user-images.githubusercontent.com/1202812/95750064-98874280-0c6a-11eb-9c5f-13c8697654fa.png">

I'd recommend adding a `margin-bottom` instead, and adding it to both the field and to the button. It should also use one of our existing spacing variables. (So perhaps something like `calc(0.5 * var(--global--spacing-vertical))`).

Reply:
Hey, @kjellr we fixed CSS as you mentioned here.